GUI/FileSpecificParametersWindow.xaml.cs

```diff
@@ -212,6 +217,7 @@ private void PopulateChoices()
             int tempMaxModsForPeptide = defaultParams.DigestionParams.MaxModsForPeptide;
             var tempPrecursorMassTolerance = defaultParams.PrecursorMassTolerance;
             var tempProductMassTolerance = defaultParams.ProductMassTolerance;
+            string tempSeparationType = defaultParams.SeparationType;
```

Comment:
This variable doesn't appear to be used for anything...

Reply:
to be honest i'm not sure how any of these are used. monkey see monkey do. set below here. 
![image](https://user-images.githubusercontent.com/16841846/72458884-19261a80-378f-11ea-9bb0-521f1d86e632.png)

Reply:
It's assigned there, but it's not being used. There is no code that calls the value of "tempSeparationType". If this was copied from something, the variable should be removed from there, too.

Reply:
i added code to use it now. When you open the gui, you see what value was in the original file. box is checked and item is selected.